### ISSUE TITLE
Create a consistent label for all our repository

### ISSUE THREAD

**Issue by Souvikns** [Tech-Phantoms/.github#10]
After #4 we can now sync labels to all our pubic repos in our org. Now would be a good time to create a suitable label system for our Organization. We are currently using a basic structure from [here](https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues). It would be nice to have the structure of a consistent label for new contributers and maintainers.

**Comment by Souvikns** [comment]
Initially, I picked up a default label system and it is really hard to understand. We also have different requirements for labels. We will be using https://github.com/marketplace/actions/label-syncer to sync labels to all our repos we will be having one YAML label file here in this repository, so in the future, if we want to update we change the file here in this repository. 

**So for now we need to find a label system for your org.**

**Comment by Souvikns** [comment]
We created a label system and you can check it [here](https://github.com/Tech-Phantoms/.github/blob/main/.github/workflows/lablel-syncer.yml), how we will be using this repository to house our labels, and then we will be using it to sync labels to all other repositories. 

> So closing this
